sdk/storage/Azure.Storage.Common/src/Shared/Errors.Clients.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Security.Authentication;
+using Azure.Core.Pipeline;
+
+namespace Azure.Storage
+{
+    // These error messages are only used by client libraries and not by the common
```

Comment:
Splitting errors into two groups allows to have fewer shared code dependencies in common and test projects.